config.yaml

```
@@ -0,0 +1,6 @@
+# MCP Server Configuration
+server:
+  host: "127.0.0.1"
+  port: 8000
+  transport: "http"
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)